src/state/routing/slice.ts

```diff
@@ -0,0 +1,99 @@
+import { createApi, fetchBaseQuery, FetchBaseQueryError } from '@reduxjs/toolkit/query/react'
+import { Protocol } from '@uniswap/router-sdk'
+// eslint-disable-next-line no-restricted-imports
+import { ChainId } from '@uniswap/smart-order-router'
+import ms from 'ms.macro'
+import qs from 'qs'
+
+import { GetQuoteResult } from './types'
+
+const protocols: Protocol[] = [Protocol.V2, Protocol.V3]
+
+const DEFAULT_QUERY_PARAMS = {
+  protocols: protocols.map((p) => p.toLowerCase()).join(','),
+  // example other params
```

Comment:
Instead of an example, can you link to the documentation/source for these query params?